ion/src/simulator/android/src/cpp/haptics.cpp

```diff
@@ -0,0 +1,46 @@
+#include "../../../shared/haptics.h"
+#include <jni.h>
+#include <SDL.h>
+
+namespace Ion {
+namespace Simulator {
+namespace Haptics {
+
+static SDL_Haptic * sSDLHaptic = nullptr;
+
+void init() {
+  if (SDL_Init(SDL_INIT_HAPTIC) == 0) {
+    sSDLHaptic = SDL_HapticOpen(0);
+    if (sSDLHaptic) {
+      if (SDL_HapticRumbleInit(sSDLHaptic) != 0) {
+        sSDLHaptic = nullptr;
+      }
+    }
+  }
+}
+
+void shutdown() {
+  if (sSDLHaptic) {
+    SDL_HapticClose(sSDLHaptic);
+  }
+}
+
```

Comment:
add a `bool isEnabled()` that is re-implemented per platform. The dummy should default to false as a failsafe (add a comment).